reduxfx-view/src/main/java/com/netopyr/reduxfx/vscenegraph/impl/patcher/NodeUtilities.java

```diff
@@ -191,7 +190,7 @@ public static Object getProperty(Object obj, String name) {
                                 .computeIfAbsent("dialog", key -> new HashMap<>())
                 ),
 
-                Case($(), Collections.emptyMap())
+                Case($(), new HashMap<>())
```

Comment:
I think this causes only a minor memory leak. Will merge the pull request and create a new issue for this.